src/models/Forecast.js

```diff
@@ -0,0 +1,105 @@
+export default class Forecast {
+  constructor(data) {
+    this.data = data;
+  }
+
+  createForecast() {
+    const forecastSection = document.querySelector('.forecast');
+    forecastSection.innerHTML = '';
+
+    Object.values(this.data).map((dayForecast, index) => {
```

Comment:
`data` is already array, `Object.values` is needless
```suggestion
    this.data.map((dayForecast, index) => {
```